### PR TITLE
Extension matching is case insensitive

### PR DESCRIPTION
**Pull request by GRiMe2D**
Some developers may prefer `.PNG` instead of `.png`.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 const mimes = require('./mimes.json');
 
 function getMime(path) {
-  const extension = path.split('.').pop();
+  const extension = path.toLowerCase().split('.').pop();
```
Comment (jahredhope) [inline index.js]: Would this make more sense?
`const extension = path.split('.').pop().toLowerCase()`
Iterating over the smaller string

File: test/index.js [hunk 2]
```diff
@@ -52,4 +52,15 @@ describe('testImageBase64', () => {
 
     assert.include(result, testImageBase64);
   });
+
+  it('case insensitive extension matching', () => {
```
Comment (jahredhope) [inline test/index.js:56]: Thank-you for adding in a unit test

**Review by jahredhope — CHANGES_REQUESTED** [review]
Thanks so much.

I'd like to see the order change as per my first comment, but I'm not hugely fussed.

I like this change, especially since I believe mimes are case-insensitive as per [RFC 2045
https://tools.ietf.org/html/rfc2045#section-2](https://tools.ietf.org/html/rfc2045#section-2)

**Comment by GRiMe2D** [comment]
Done.

**Review by jahredhope — APPROVED** [review]

**Pull request merged**